pywb/warcserver/index/fuzzymatcher.py

```diff
@@ -194,6 +194,11 @@ def match_general_fuzzy_query(self, url, urlkey, cdx, rx_cache):
         check_query = False
         url_no_query, ext = self.get_ext(url)
 
+        # don't fuzzy match to 204
+        if cdx.get('status') == '204':
+            if '__pywb_method=options' in cdx['urlkey']:
```

Comment:
do not fuzzy much if 204 and method options?

Reply:
Yes, if we encounter an OPTIONS response with 204, don't include it as a match.
Mostly for warcprox WARCs which write OPTIONS requests with 204.